src/Provider/Doctrine/Auditing/Transaction/AuditTrait.php

```diff
@@ -75,6 +76,10 @@ private function value(EntityManagerInterface $entityManager, Type $type, $value
             return null;
         }
 
+        if ($value instanceof BackedEnum) {
```

Comment:
I'd double check PHP version + remove the `use` statement line 7 + use the FQCN here (`\BackedEnum`) here in the test

Reply:
Added double check and FQCN, but previously I run php-cs-fixer that removed leading slash

Reply:
@DamienHarper should I run cs-fixer? It will remove FQCN for BackedEnum and will touch some lines I never changed

Reply:
You're right, let csfixer fix the file and commit it back. It shouldn't be problematic since you previously check PHP version. That way, it should go further in the CI pipeline (will still fail with PHPStan and PHP 7.4 but that's another story :))

Reply:
cs-fixer done